controller/src/main/java/io/pravega/controller/server/AuthResourceRepresentation.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server;
+
+import io.pravega.common.Exceptions;
+
+/**
+ * A utility class with methods for preparing string representations of auth-protected resources.
+ */
+public class AuthResourceRepresentation {
+
+    public static String ofScopes() {
```

Comment:
Please add Javadoc for each of these methods. I cannot tell what they're supposed to do just from their names.

Reply:
Sure. Have done so now.